### PR TITLE
Merging common code bases StringsHelper

### PR DESCRIPTION
**Pull request by DavoudEshtehari**
- Related issue #1261 
- Common type: **StringsHelper**

### CODE REVIEW

**Review by lcheunglci — APPROVED** [review]
Looks good to me

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj [hunk 1]
```diff
@@ -330,6 +330,9 @@
     <Compile Include="..\..\src\Resources\ResDescriptionAttribute.cs">
       <Link>Resources\ResDescriptionAttribute.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\Resources\StringsHelper.cs" >
+      <Link>Resources\StringsHelper.Common.cs</Link>
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:334]: Can we link it as StringsHelper.cs?

File: src/Microsoft.Data.SqlClient/src/Resources/StringsHelper.cs [hunk 2]
```diff
@@ -7,7 +7,12 @@
 using System.Resources;
 using System.Threading;
 
-namespace Microsoft.Data
+namespace
+#if NETFRAMEWORK
+    Microsoft.Data
+#else
+    System
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/src/Resources/StringsHelper.cs:14]: Can't it work with Microsoft.Data namespace in .NET Core?

File: src/Microsoft.Data.SqlClient/src/Resources/StringsHelper.cs [hunk 3]
```diff
@@ -16,12 +21,16 @@ internal partial class StringsHelper : Strings
 
         internal StringsHelper()
         {
+#if NETFRAMEWORK
             _resources = new ResourceManager("SqlClient.Resources.Strings", GetType().Assembly);
+#else
+            _resources = new ResourceManager("Microsoft.Data.SqlClient.Resources.Strings", GetType().Assembly);
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/src/Resources/StringsHelper.cs:27]: This can be unified is namespace is unified, please try that change.

**Review by cheenamalhotra — APPROVED** [review]

**Review by JRahnama — APPROVED** [review]

**Pull request merged**